docs/product_release.md

```diff
@@ -0,0 +1,18 @@
+# Product Release
+
+## Artifacts
+
+- Container images
+- Helm Chart
+- GitHub release (Changelog + Sourcecode)
+
+## Versioning
+
+- Semantic Versioning
+
+## Quality Assurance
+
+- Defined in TRGs
+- Responsibility of every contributor
+- (Additonally verified in quality gate process. TBD, if kept after consortia)
```

Comment:
Typo in Additonally but mainly I'd not mention consortia, may just:
Currently still verified in a Quality Gate process. TBD if this process will continue.